src/MissionManager/ComplexMissionItem.cc

```diff
@@ -396,6 +396,24 @@ void ComplexMissionItem::_intersectLinesWithPolygon(const QList<QLineF>& lineLis
     }
 }
 
+/// Adjust the line segments such that they are all going the same direction with respect to going from P1->P2
+void ComplexMissionItem::_adjustLineDirection(const QList<QLineF>& lineList, QList<QLineF>& resultLines)
+{
+    for (int i=0; i<lineList.count(); i++) {
+        const QLineF& line = lineList[i];
+        QLineF adjustedLine;
+
+        if (line.angle() > 180.0) {
```

Comment:
But that also says that grid angle also informs initial entry point. I was thinking that wasn't the case, but I can see now how it may be. What's the concensus on that?